test/functional/fixtures/runner/upload/upload.test.js

```diff
@@ -0,0 +1,19 @@
+'@fixture Upload';
+'@page ./index.html';
+
+
+'@test'['Upload single file test - should fail'] = {

```

Comment:
I suppose we should name it like `Upload not existent file - should fail` or something like this.
@MargaritaLoseva please check all strings in this PR please